README.md

```diff
@@ -27,31 +27,43 @@ vi dev.env
 ## Deploying with Docker Compose
 
 ```bash
+# wi2node-ctl options
+python3 wis2node-ctl.py
+
+# view docker config
+python3 wis2node-ctl.py config
+
 # build local images
-make build
+python3 wis2node-ctl.py build
 
 # start system
-make up
+python3 wis2node-ctl.py start
 
 # login to main wis2node container
-make login
+python3 wis2node-ctl.py login
+
+# restart containers
+python3 wis2node-ctl.py restart
+
+# view upstatus
+python3 wis2node-ctl.py status -a
```

Comment:
what does the -a flag do? It may be worth adding a comment into the readme.

Reply:
I've accepted and pulled as we'll want a comprehensive review of the readme at a later date.

Reply:
@david-i-berry we are at a weird point where the README is split as a stand-in for actual documentation while also being the quickstart. It is approaching enough complexity that the documentation might need to be separate from a quickstart. The -a flag was included because I was trying to have the test workflow mirror the README.